powermonitor/main.js

```diff
@@ -65,19 +65,29 @@ module.exports = function setupPowerMonitorMain(jitsiMeetWindow) {
     }
 
     ipcMain.on(POWER_MONITOR_QUERIES_CHANNEL, (source, { id, data }) => {
+        const { powerMonitor } = electron;
+
         switch(data.type) {
             case METHODS.queryIdleState:
-                electron.powerMonitor.querySystemIdleState(
-                    data.idleThreshold,
-                    idleState => {
-                        systemIdleResult(id, idleState);
-                    });
+                if (typeof powerMonitor.getSystemIdleState === 'function') { // electron 5+
+                    systemIdleResult(id, powerMonitor.getSystemIdleState(data.idleThreshold));
+                } else { // electron 4 or older
```

Comment:
Question: do we need to support Electron < 5?

Reply:
We do unfortunately! We even have production use case for this!

We could probably just leave it unsupported right now and tell anybody who wants to use electron 4 to use the old release but I'm worried that we may need to push fixes for the old releases if we go down this road. This will be very hard for maintenance.... Supporting it for now looks easier...

Also upgrading to electron 5 with node 12 for projects with a lot of native add-ons turns out to be not so trivial task. So I guess it is good to support it for some time in order to give consumers more time to upgrade electron. Currently this is the only part of the code which needs adjustment.